dhis-2/dhis-api/src/main/java/org/hisp/dhis/period/Period.java

```diff
@@ -375,8 +375,7 @@ private boolean objectEquals( Period other )
     @Override
     public String toString()
     {
-        return "[" + (dateField == null ? "DEFAULT" : dateField) + " | "
-            + (periodType == null ? "" : periodType.getName() + ": ") + startDate + " - " + endDate + "]";
+        return getIsoDate();
```

Comment:
Why did you make this change? Was it because `toString` was ugly, unintuitive and overly-complicated? If so I agree with these reasons. However if `dateField` is present then `toString` won't show the difference whereas `equals` will. I'm not sure this is good. How about:

        return getIsoDate() + (isDefault() ? "" : ":" + dateField);

(This is more like the how the dateField was first parsed in `IsoPeriodHolder`.)

Reply:
I found myself having to expand into the details of an period object to read what period it was referring to so I thought this can be more readable/informative simply by returning its string form, the ISO date. 

Added the dateField as you suggested.